pages/linux/grub-install.md

```diff
@@ -0,0 +1,12 @@
+# grub-install
+
+> Install GRUB to a device.
+> More information: <https://www.gnu.org/software/grub/manual/grub/html_node/Installing-GRUB-using-grub_002dinstall.html>.
+
+- Install GRUB on a BIOS system:
+
+`grub-install --target={{i386-pc}} {{/path/to/device}}`
```

Comment:
The patch could be relative, like `mkfs`, don't start with `/`:
```suggestion
`grub-install --target={{i386-pc}} {{path/to/device}}`
```

Reply:
An honest mistake

Reply:
@navarroaxel So when the path must be absolute, it is to be lead with `/`? Just noticed one in `transmission-remote` and decided to ask before opening a pull request.

Reply:
@Managor yeah, in general we use `path/to/thing` for relative paths, and `/path/to/thing` for absolute paths. Unless we're on a windows-specific page I'd guess, in which case absolute paths would probably be represented by something like `Z:\path\to\file` or something (but that hasn't yet come up as far as I'm aware).